index.css

```diff
@@ -5,6 +5,130 @@
   margin-left: 0;
 }
 
+#modulo {
+  display: none;
+  position: fixed;
+  backdrop-filter: blur(8px);
+  right: 0;
+  left: 0;
+  top: 0;
+  bottom: 0;
+  z-index: 200;
+}
+
+#modulo .content {
+  border: 1px solid #ff6b00;
+  background-color: white;
+  margin-top: 40px;
+  margin-bottom: 40px;
+  margin-left: 26px;
+  margin-right: 26px;
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+#modulo .heading {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+}
+
+#modulo .close {
+  padding: 14px;
+  color: #67798e;
+  height: 20px;
+}
+
+footer h2 {
+  width: 327px;
+  font-family: "Inter", sans-serif;
+  text-align: center;
+  font-size: 32px;
+  font-weight: bold;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.myresent h2 {
+  display: flex;
+  justify-content: center;
+}
+
+#modulo .heading h2 {
+  color: #172b4d;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 28px;
+  line-height: 44px;
+  padding-left: 10px;
+}
+
+#modulo .image {
+  width: 311px;
+  object-fit: cover;
+  min-height: 100px;
+}
+
+#modulo .description {
+  color: #344563;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  margin-top: 24px;
+  flex-shrink: 1;
+  min-height: 0;
+  overflow: auto;
+}
+
+.pop button {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  color: #3c3a39;
+}
+
+footer ul {
+  list-style: none;
+}
+
+.pop ul {
+  height: 40px;
+  list-style: none;
+  display: flex;
+  padding: 0 0 0 0;
+}
```

Comment:
- [ ] It would be nicer to work more on the design of the languages list to make your design matches the Figma design.
- Your design:
<img width="266" alt="image" src="https://user-images.githubusercontent.com/77190203/162259397-271a6ef6-df89-4c05-9631-6b0e4a38485d.png">

- Figma design:
<img width="277" alt="image" src="https://user-images.githubusercontent.com/77190203/162259507-c8f23066-6619-4724-8a6c-835d5dc08224.png">

Reply:
Thank you very much for  the CR